### PR TITLE
Use native `JSONBField`s everywhere (instead of `JSONField`s)

### PR DESCRIPTION
**Pull request by noliveleger**

### CODE REVIEW

File: kpi/models/import_export_task.py [hunk 1]
```diff
@@ -82,8 +82,9 @@ class Meta:
     )
 
     user = models.ForeignKey('auth.User', on_delete=models.CASCADE)
-    data = JSONField()
-    messages = JSONField(default={})
+    # ToDo validate whether both fields below need to be changed to jsonbfield
```
Comment (jnm) [inline kpi/models/import_export_task.py]: `data` is definitely an improvement as shown in your `filter()` change below. as for `messages`, why not?

File: kpi/models/import_export_task.py [hunk 2]
```diff
@@ -600,9 +601,8 @@ def _run_task(self, messages):
     def _filter_by_source_kludge(queryset, source):
         """
         A disposable way to filter a queryset by source URL.
-        TODO: make `data` a `JSONBField` and use proper filtering
         """
-        return queryset.filter(data__contains=source)
+        return queryset.filter(data__source=source)
```
Comment (jnm) [inline kpi/models/import_export_task.py]: great, but it seems like `_filter_by_source_kludge()` is used only in `kpi/tests/test_mock_data_exports.py`. maybe we trash this method and use `filter(data__source=source)` directly in the tests? (or at least remove `_kludge` from the method name :smile:)

File: kpi/views/v1/export_task.py [hunk 3]
```diff
@@ -33,9 +33,7 @@ def get_queryset(self, *args, **kwargs):
             return queryset
         if q.startswith('source:'):
             q = remove_string_prefix(q, 'source:')
-            # This is exceedingly crude... but support for querying inside
-            # JSONField not available until Django 1.9
-            queryset = queryset.filter(data__contains=q)
+            queryset = queryset.filter(data__source=q)
```
Comment (jnm) [inline kpi/views/v1/export_task.py]: awesome

File: kpi/models/import_export_task.py (hunk 2 above)
Comment (noliveleger) [reply kpi/models/import_export_task.py]: indeed

File: kpi/migrations/0015_assetversion.py [hunk 4]
```diff
@@ -69,7 +70,7 @@ class Migration(migrations.Migration):
         migrations.AlterField(
             model_name='asset',
             name='summary',
-            field=JSONBField(default=dict, null=True),
+            field=jsonfield.fields.JSONField(default=dict, null=True),
```
[the remark this answers is not in the document]
Comment (noliveleger) [reply kpi/migrations/0015_assetversion.py]: that's an oversight.

**Pull request merged**